lib/CL/devices/common.h

```diff
@@ -26,7 +26,7 @@
 #define POCL_COMMON_H
 
 #include "pocl_cl.h"
-#include "dev_image.h"
+#include "_kernel_c.h"
```

Comment:
I've copied the required `#define`s to `dev_image.h`.

Is the `cl_int` type also defined in the kernel context? In that case the types in `include/pocl_types.h` could be changed to `cl_int` and the `#define`s can be moved to that file. That way `dev_images.h` can be removed.